web-sdk/v3.md

```diff
@@ -183,6 +186,12 @@ const label = {
     <td>function</td>
     <td>Function to be called when a Card verification fails. Receives an <a href="#error-object">error object</a> as argument including the cardId.</td>
   </tr>
+  <tr>
+    <td><code>onUserCancelledCallback</code></td>
+    <td>function</td>
+    <td>Function to be called when the user closes the SDK.  
```

Comment:
I think it should be "when the user cancels the card connection process at any point." Again, "closing the SDK" can be interpreted as "closing when finishing or completing the process" as well.

Reply:
"The user closes the SDK" implies an action from the user to close the SDK and not just a consequence of ending the flow. I would agree with you with something like "whenever the SDK is closed".

Reply:
I have contacted @krisztina-nagy for assistance on this topic.

Reply:
What if we write "when the user closes the SDK without finishing the card enrollment." ? Would that be correct?

Reply:
That would not be correct @krisztina-nagy, because the user can enroll the card and cancel at the card verification step. Why is it not ok to use the "cancel" semantics? Not sure why we cling to using "close"/"closing".

Reply:
Oh, ok, I understand now. I don't mind "cancel", just tried to avoid explaining something with the same word. But that's secondary, so we can go ahead with "cancel".